res/controllers/common-controller-scripts.js

```diff
@@ -368,6 +368,7 @@ bpm.tapButton = function(deck) {
 // ----------------- Common regular expressions --------------------------
 script.samplerRegEx = /\[Sampler(\d+)\]/ ;
 script.channelRegEx = /\[Channel(\d+)\]/ ;
+script.channelRegExExact = /^\[Channel(\d+)\]$/ ;
```

Comment:
adding a new regex rather than modifying the existing one because there may be a use case for matching the [ChannelX] part of EQ rack or QuickEffectRack groups